ts/store/reducers/entities/messages/transformers.ts

```diff
@@ -101,7 +102,9 @@ const getEUCovidCertificate = ({
  * @param messageFromApi
  */
 export const toUIMessageDetails = (
-  messageFromApi: CreatedMessageWithContentAndAttachments
+  messageFromApi:
+    | CreatedMessageWithContentAndAttachments
+    | CreatedMessageWithContent
```

Comment:
Not sure whether we need the union type: `CreatedMessageWithContentAndAttachments` should be the equivalent of a supertype of `CreatedMessageWithContent`? Maybe we can simplify it.

Reply:
super! it works!
22770a206076ccdb31b19fe6803e0e61af383510